### PR TITLE
[Community] Add vLLM Discord server

### PR DESCRIPTION
**Pull request by zhuohan123**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,13 +10,14 @@ Easy, fast, and cheap LLM serving for everyone
...
 <p align="center">
-| <a href="https://vllm.readthedocs.io/en/latest/"><b>Documentation</b></a> | <a href="https://vllm.ai"><b>Blog</b></a> | <a href="https://arxiv.org/abs/2309.06180"><b>Paper</b></a> | <a href="https://github.com/vllm-project/vllm/discussions"><b>Discussions</b></a> |
+| <a href="https://vllm.readthedocs.io/en/latest/"><b>Documentation</b></a> | <a href="https://vllm.ai"><b>Blog</b></a> | <a href="https://arxiv.org/abs/2309.06180"><b>Paper</b></a> | <a href="https://discord.gg/jz7wjKhh6g"><b>Discord</b></a> |
 
 </p>
 
 ---
 
 *Latest News* 🔥
+- [2023/10] We created our [Discord server](https://discord.gg/jz7wjKhh6g) for vLLM! Join us to discuss vLLM and LLM serving! We will also post the latest updates on vLLM there.
```
Comment (WoosukKwon) [inline README.md:20]: Why is it Oct?

Comment (zhuohan123) [reply README.md]: I should stop working in late night when my brain becomes numb and starts to make very silly mistakes...

File: README.md [hunk 2]
```diff
@@ -10,13 +10,14 @@ Easy, fast, and cheap LLM serving for everyone
...
 <p align="center">
-| <a href="https://vllm.readthedocs.io/en/latest/"><b>Documentation</b></a> | <a href="https://vllm.ai"><b>Blog</b></a> | <a href="https://arxiv.org/abs/2309.06180"><b>Paper</b></a> | <a href="https://github.com/vllm-project/vllm/discussions"><b>Discussions</b></a> |
+| <a href="https://vllm.readthedocs.io/en/latest/"><b>Documentation</b></a> | <a href="https://vllm.ai"><b>Blog</b></a> | <a href="https://arxiv.org/abs/2309.06180"><b>Paper</b></a> | <a href="https://discord.gg/jz7wjKhh6g"><b>Discord</b></a> |
 
 </p>
 
 ---
 
 *Latest News* 🔥
+- [2023/09] We created our [Discord server](https://discord.gg/jz7wjKhh6g) for vLLM! Join us to discuss vLLM and LLM serving! We will also post the latest updates on vLLM there.
```
Comment (WoosukKwon) [inline README.md:20]: nit: I feel vLLM is repeated too much? 😅 
```suggestion
- [2023/09] We created our [Discord server](https://discord.gg/jz7wjKhh6g)! Join us to discuss vLLM and LLM serving! We will also post the latest updates on vLLM there.
```

Comment (zhuohan123) [reply README.md]: Just fixed!

**Review by WoosukKwon — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**